DEVELOPMENT.md

```diff
@@ -0,0 +1,50 @@
+## About 
+This document describes development-related topics of pgn-tactics-generator. 
+pgn-tactics-generator is a python application dedicated to creating chess puzzles/tactics from a pgn file.
+See [readme](./README.md) file for more details.
+
+
+# Dependencies
+
+This script requires the [Requests](https://docs.python-requests.org/) and [python-chess](https://python-chess.readthedocs.io/)
+libraries to run, as well as a copy of *Stockfish*.
+Is recommended that you use Python 3 and pip3. 
+It should work with Python 2.7 and pip (probably you will need to install futures `pip install futures` )
```

Comment:
Not sure if we support (or should support) python 2.7 anymore. So probably is best to just remove those lines and do the documentation based on python3 and pip3. What do you think?

Reply:
I've created #28